inspirehep/modules/migrator/manage.py

```diff
@@ -106,6 +106,11 @@ def populate(records, collections, file_input=None, remigrate=False,
 
 
 @manager.command
+def count_citations():
```

Comment:
Can you add a call to this command at the end of `install.sh` so that it's going to be part of the Travis build? You can also mention running it in the documentation in the install guide.

Also you can then add a simple test that assesses e.g. that record 1124337 has 46 citations :-)